daemon/listener/listener.js

```diff
@@ -185,13 +190,26 @@ async function handleLog(log, rule, contractVersion, context) {
   }
 
   const json = JSON.stringify(output, null, 2)
+  if (context.config.verbose) {
+    console.log(json)
+    console.log('\n----\n')
+  }
+
+  //TODO(franck): remove binary data from pictures in a proper way.
+  let listing = output.related.listing
+  delete listing.ipfsData.data.pictures
+
+  if (context.config.search) {
+    await search.indexListing(listing.id, listing.ipfsData.data)
+  }
+
+  if (context.config.db) {
+    await db.insertListing(listing.id, listing.ipfsData.data)
+  }
 
   if (context.config.webhook) {
-    process.stdout.write('\n-- WEBHOOK to ' + context.config.webhook + ' --\n')
+    console.log('\n-- WEBHOOK to ' + context.config.webhook + ' --\n')
     await postToWebhook(context.config.webhook, json)
-  } else {
```

Comment:
I still think we need some way to send to stdout - it's really handy for debugging the listener, as well as debugging other systems, and it's just a handy feature to do unixy things with.

Reply:
Yes, agreed. The --verbose option will dump the log to stdout.